src/Drivers/SymfonyTinkerwellDriver.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+use Tinkerwell\ContextMenu\Label;
+use Tinkerwell\ContextMenu\OpenURL;
+use Tinkerwell\ContextMenu\SetCode;
+use Tinkerwell\ContextMenu\Submenu;
+use Symfony\Component\HttpFoundation\Request;
+
+class SymfonyTinkerwellDriver extends TinkerwellDriver
+{
+    public function canBootstrap($projectPath)
+    {
+        return file_exists($projectPath . '/public/index.php') &&
```

Comment:
This is how the Symfony valet driver detects if Symfony is installed: https://github.com/laravel/valet/blob/master/cli/drivers/SymfonyValetDriver.php#L15-L17
Maybe we can just use this?

Reply:
I'm running the Symfony Skelaton project and it doesn't have the `web` folder at all.. perhaps we can use everything else?

```php
 (file_exists($sitePath.'/app/AppKernel.php')) || (file_exists($sitePath.'/public/index.php')) &&
 (file_exists($sitePath.'/src/Kernel.php'))
```

Reply:
@mpociot I'm trying to test this locally with a Symfony skeleton project, but for some reason, it is not loading. I'm using the latest 2.2 version of Tinkerwell. I created the folder `.tinkerwell` and put this class into it. Am I missing something?

Like @Braunson is saying, there is no web folder, only the public one.

Reply:
Okay, then let's just stick to the detection that is in here.

Is there anything else that needs to be modified then?

Reply:
@mpociot seems good to me! 👍